packages/libpluto/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
+TERMUX_PKG_DESCRIPTION="Shared library for the Pluto interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION="0.9.4"
+TERMUX_PKG_REVISION=2
+TERMUX_PKG_SRCURL=https://github.com/PlutoLang/Pluto/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a4fae052a9ad47e29a6767047c3e55ab5c887486d14e39248657ff43b45875c
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_BUILD_DEPENDS="readline"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_configure() {
+	export PHPENV_ROOT="${TERMUX_PKG_CACHEDIR}/phpenv"
+	export PATH="${PHPENV_ROOT}/bin:${PATH}"
+	curl -L https://raw.githubusercontent.com/phpenv/phpenv-installer/master/bin/phpenv-installer | bash
```

Comment:
Couple things here:
- Why are we using this specific 3rd party tool?
- If we do wanna use this one, please pin the download to a specific commit.
- Use `termux_download` to retrieve the file and verify its integrity via checksum.